ports/common/include/atomic/iot_atomic_gcc.h

```diff
@@ -140,7 +140,14 @@ static FORCE_INLINE uint32_t Atomic_Add_u32( uint32_t volatile * pAugend,
     /* coverity[misra_c_2012_rule_8_5_violation] */
     /* coverity[misra_c_2012_rule_17_3_violation] */
     /* coverity[caretline] */
+#ifdef CBMC
+#    pragma CPROVER check push
```

Comment:
I think @markrtuttle had a way to resolve CBMC's issues with this file without modifying it?

Reply:
@gordonwang0 In fact, the current version of the proof harness already resolve this issue without the `#pragma`, but I added it anyway in response to this [comment](https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/805#discussion_r381370884) from @danielsn, do you want to keep the `#pragma` or remove it from the file?

Reply:
I think it would be best to not modify this file.

Reply:
Ok, I removed it.